code/services/ExampleGetDataset/ExampleGetDataset.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Fetches the given dataset in a project 
+ * @param {Object} req
+ * @param {callback} resp - Dataset Resource object is returned!
+ */
+function ExampleGetDataset(req, resp){
+    var options = {
+        authToken: 'YOUR_BIGQUERY_AUTH_TOKEN' //(is the API key for Google BigQuery Service, https://cloud.google.com/bigquery/docs/authorization)
+        , projectID: 'gentle-impulse-161804'
+        , datasetID: 'babynames'
+        //(optional parameters)
```

Comment:
inline comments in object declarations are discouraged

Reply:
Done.